Dalamud/Plugin/Internal/Profiles/PluginManagementCommandHandler.cs

```diff
@@ -216,6 +216,11 @@ void Continuation(Task t, string onSuccess, string onError)
             
             this.chat.Print(onSuccess);
         }
+
+        if (operation is PluginCommandOperation.Toggle)
+        {
+            return HandlePluginOperation(workingPluginId, plugin.State == PluginState.Loaded ? PluginCommandOperation.Disable : PluginCommandOperation.Enable);
+        }
```

Comment:
Close - you can completely avoid the recursion call entirely, since the switch handles it:

```suggestion
        if (operation is PluginCommandOperation.Toggle)
        {
            operation = (plugin.State == PluginState.Loaded ? PluginCommandOperation.Disable : PluginCommandOperation.Enable));
        }
```

Reply:
oh oh my minds mess